src/compiler/crystal/tools/formatter.cr

```diff
@@ -4577,8 +4573,8 @@ module Crystal
       raise "expecting keyword #{keywords.join " or "}, not `#{@token.type}, #{@token.value}`, at #{@token.location}" unless keywords.any? { |k| @token.keyword?(k) }
     end
 
-    def check(token_type)
-      raise "expecting #{token_type}, not `#{@token.type}, #{@token.value}`, at #{@token.location}" unless @token.type == token_type
+    def check(*token_types)
+      raise "expecting #{token_types.join " or "}, not `#{@token.type}, #{@token.value}`, at #{@token.location}" unless token_types.includes? @token.type
```

Comment:
Splitting this from one line is overdue now 😄

Reply:
Just change the postfix `unless` to normal `unless` and it's readable enough.